generators/chipyard/src/main/scala/config/RadianceConfigs.scala

```diff
@@ -0,0 +1,157 @@
+package chipyard
+
+import chipyard.config.AbstractConfig
+import chipyard.stage.phases.TargetDirKey
+import freechips.rocketchip.devices.tilelink.BootROMLocated
+import freechips.rocketchip.resources.BigIntHexContext
+import freechips.rocketchip.subsystem._
+import org.chipsalliance.cde.config.Config
+import radiance.subsystem.RadianceGemminiDataType
+
+// ----------------
+// Radiance Configs
+// ----------------
+
+// aliases for virgo
+class VirgoConfig extends RadianceClusterConfig
+class VirgoFP16Config extends RadianceFP16ClusterConfig
+class VirgoHopperConfig extends Radiance4CFP16ClusterConfig
+class VirgoFlashConfig extends RadianceClusterConfig
+class VirgoSynConfig extends RadianceClusterSynConfig
+class VirgoFP16SynConfig extends RadianceFP16ClusterSynConfig
+class VirgoHopperSynConfig extends Radiance4CFP16ClusterSynConfig
+
+class RadianceBaseConfig extends Config(
+  // NOTE: when changing these, remember to change NUM_CORES/THREADS/WARPS in
+  // the verilog source as well!
+  new radiance.subsystem.WithSimtConfig(nWarps = 8, nCoreLanes = 8, nMemLanes = 8, nSrcIds = 32) ++
+  new chipyard.config.WithSystemBusWidth(bitWidth = 256) ++
+  new WithExtMemSize(BigInt("80000000", 16)) ++
```

Comment:
Nit: leaving the full package paths here is cleaner I think (including chipyard.AbstractConfig)